src/metatrain/cli/export.py

```diff
@@ -68,9 +67,11 @@ def _prepare_export_model_args(args: argparse.Namespace) -> None:
     for key in original_keys:
         if key not in keys_to_keep:
             args.__dict__.pop(key)
+    if args.__dict__.get("output") is None:
+        args.__dict__["output"] = Path.cwd() / (Path(path).stem + ".pt")
```

Comment:
I think you don't need the absolute path to the working directory. I will be written by default to the current cwd.